docs/tutorial.rst

```diff
@@ -1250,63 +1235,34 @@ unique string ID. Here, we'll add the site in the middle of the contig, with ID
     contains a single site mutation, the single site mutation will be
     "overwritten" and an error will be raised in simulation.
 
-Next, we will set up the "extended events" which will modify the demography.
-The first extended event is the origination of the selected mutation, which
-will occur in a random individual from the first population (id 0), 1000
-generations ago.
+Next, we will set up the "SLiM events" which will modify the demography.  This
+is done through :func:`stdpopsim.ext.selective_sweep`, which represents a general model
+for a mutation that is beneficial over a time interval within a single
```

Comment:
```suggestion
for a mutation that is beneficial within a single
```
removing the time interval bit, because we aren't yet allowing for time inhomogenous selection and I think this is clearer